index.html

```diff
@@ -0,0 +1,76 @@
+<html lang="en-US">
+
+<head>
+  <title> mon appétit </title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <header class="header-background">
+    <h1 class="main-text"> mon appétit </h1>
+    <nav class="navbar">
+      <a href="index.html">HOME</a>
+      <a href="about.html">ABOUT</a>
+    </nav>
+  </header>
+  <main>
```

Comment:
I don't have any big critiques about your code itself, but I did notice that you had quite a few different font sizes and styles on this page. It definitely is a nitpick tho, so if you like it how it is don't worry about changing it :)